README.md

```diff
@@ -328,9 +328,9 @@ Native desktop applications for managing and montoring docker hosts and clusters
 
 - [Captain](https://getcaptain.co/) - Manage containers from the MacOSX menu bar by [@RickWong](https://github.com/rickwong)
 - [Dockeron](https://github.com/dockeron/dockeron) - A project built on Electron + Vue.js for Docker on desktop. [@fluency03](https://github.com/fluency03)
+- [DockerStacks](https://github.com/sfx101/docker-stacks) - Local LAMP/LEMP Development Studio [@sfx101](https://github.com/sfx101)
```

Comment:
could also mention that this is not an open source program (closed source). It is confusing to see a github project without the source. Or just add the website instead of the repo.

Reply:
Yes, it's closed source at the moment. I've added the link to the site. Please let me know if that works